android/AndroidManifest.xml

```diff
@@ -43,6 +43,18 @@
       android:theme="@style/AppTheme"
       android:label="@string/app_name">
 
+        <intent-filter>
+            <action android:name="android.intent.action.VIEW" />
+            <category android:name="android.intent.category.DEFAULT" />
+
+            <data android:scheme="content" />
+            <data android:mimeType="*/*"/>
+            <data android:pathPattern=".*\\.zim(aa|)" />
+            <data android:pathPattern=".*\\..*\\.zim(aa|)" />
+            <data android:pathPattern=".*\\..*\\..*\\.zim(aa|)" />
+            <data android:pathPattern=".*\\..*\\..*\\..*\\.zim(aa|)" />
+        </intent-filter>
+
```

Comment:
This is intercepting too much. If I get a gmail notification and click on it I get asked whether I want to open in our app. This should not happen.